cli/src/commands/init/index.ts

```diff
@@ -50,6 +78,10 @@ export default class Init extends BaseCommand<typeof Init> {
     }),
     schema: Flags.string({
       description: 'Initializes a new database or updates an existing one with the given schema'
+    }),
+    packageManager: Flags.string({
```

Comment:
Almost forgot! This should be `package-manager` if we follow CLI flag naming conventions